liquibase/liquibase.github.com
# 3.8.6release

PLEASE PUSH THIS MORNING FOR RELEASE

## What I Did
Completed several changes for the 3.8.6 release announcement.
## How To Test It
Check to ensure the new blog shows up, the banner is updated, and other text updates show up.
